### DISCUSSION TITLE
Getting an error on api.create_tweet. Seems like it's deprecated?

### DISCUSSION THREAD

**Discussion by realrodri** [tweepy/tweepy#1779] [Questions] [answered]
Error comment: `AttributeError: 'API' object has no attribute 'create_tweet'. Did you mean: 'create_list'?`

Code I have on that line: `response = api.create_tweet(in_reply_to_tweet_id='tweetNumber', text='myText')`

No idea what's causing the error, if I'm following the docs on the Tweepy documentation :/

**Comment by Harmon758** [comment, marked as answer]
As the error says, [`API`](https://tweepy.readthedocs.io/en/v4.4.0/api.html#tweepy.API) doesn't have a `create_tweet` attribute/method.
You're probably looking for [`API.update_status`](https://tweepy.readthedocs.io/en/v4.4.0/api.html#tweepy.API.update_status) or [`Client.create_tweet`](https://tweepy.readthedocs.io/en/v4.4.0/client.html#tweepy.Client.create_tweet), depending on whether you're using Twitter API v1.1 or v2.
